### PR TITLE
Adding Azure Update Management Workbook to the Update Management gallery

### PR DESCRIPTION
**Pull request by swjai**
## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [ ] post a screenshot of templates and/or gallery changes
* [ ] ensure your template has a corresponding gallery entry in the gallery folder
* [ ] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [ ] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [ ] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [ ] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [ ] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [ ] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

**Review by assing — APPROVED** [review]

**Comment by yves-chan** [comment]
Please fix this error: 

 1) Validating Workbooks...
       Verifying .workbook files:
     Uncaught AssertionError: Found duplicate key: items.Assessment.content.items.query6.content.title in template: ./Workbooks/Azure Update Management/Azure Update Management.workbook. To fix this error, change the step name or id

You can look at the details of the Template validation to see what failed during the build process

File: Workbooks/Azure Update Management/Azure Update Management.workbook [hunk 1]
```diff
@@ -0,0 +1,668 @@
...
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "Heartbeat \r\n| extend IsEnabled = iff(Solutions has 'updates', 'Yes', 'No') \r\n| where Computer contains '{name}'\r\n| summarize arg_max(TimeGenerated, OSType, IsEnabled) by _ResourceId",
+              "size": 0,
+              "title": "Machines reporting to Log analytics with Updates solution enabled vs not enabled.",
+              "timeContext": {
+                "durationMs": 86400000
```
Comment (yves-chan) [inline Workbooks/Azure Update Management/Azure Update Management.workbook:130]: Will you need a timerange parameter for this workbook or will it be defaulted a time for all custoemrs?

File: Workbooks/Azure Update Management/Azure Update Management.workbook [hunk 2]
```diff
@@ -0,0 +1,668 @@
...
...mputersCount=sumif(computersBySeverity, WorstMissingUpdateSeverity==-1), computersNeedCriticalUpdatesCount=sumif(computersBySeverity, WorstMissingUpdateSeverity==4), computersNeedSecurityUpdatesCount=sumif(computersBySeverity, WorstMissingUpdateSeverity==2), computersNeedOtherUpdatesCount=sumif(computersBySeverity, WorstMissingUpdateSeverity==1), upToDateComputersCount=sumif(computersBySeverity, WorstMissingUpdateSeverity==0)\r\n| summarize assessedComputersCount=sum(assessedComputersCount), computersNeedCriticalUpdatesCount=sum(computersNeedCriticalUpdatesCount),  computersNeedSecurityUpdatesCount=sum(computersNeedSecurityUpdatesCount), computersNeedOtherUpdatesCount=sum(computersNeedOtherUpdatesCount), upToDateComputersCount=sum(upToDateComputersCount), notAssessedComputersCount=sum(notAssessedComputersCount)\r\n| extend allComputersCount=assessedComputersCount+notAssessedComputersCount\r\n| project machinePack = pack(\"Assessed machines\", assessedComputersCount, \"Not assessed machines\", notAssessedComputersCount)\r\n| mv-expand machinePack\r\n| extend machine = tostring(bag_keys(machinePack)[0])\r\n| extend count_ = tolong(machinePack[machine])\r\n| project machine, count_ ",
```
Comment (yves-chan) [inline Workbooks/Azure Update Management/Azure Update Management.workbook:180]: Same here, I guess general question for the whole workbook: Is timerange parameter needed?

File: Workbooks/Azure Update Management/Azure Update Management.workbook [hunk 3]
```diff
@@ -0,0 +1,668 @@
...
+                  },
+                  {
+                    "columnId": "count_",
+                    "label": "Machines"
+                  }
+                ]
+              }
+            },
+            "customWidth": "50",
+            "name": "query - 6"
```
Comment (yves-chan) [inline Workbooks/Azure Update Management/Azure Update Management.workbook:443]: nit: can you look through all the name fields and rename the items to something meaningful and descriptive?

**Comment by gardnerjr** [comment]
closing ancient uncompleted pull requests, this is more than a year untouched

**Pull request closed without merging**